.github/workflows/test.yml

```diff
@@ -1,36 +1,27 @@
 name: Build and Run Tests
 
-on:
-  pull_request:
-    branches:
-      - master
+on: [ pull_request ]
 
 jobs:
   build_and_test:
     name: Build and test
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [ 16, 18 ]
+#        node: [ 16, 18 ]
+        node: [ 16, 17 ]
```

Comment:
I think that 17 should not be part of the testing.

Reply:
It's not, it was just a test.